Tests/UnitTests/Mocks/Sessions+Services/Storage/MockDefaultsStore.swift

```diff
@@ -13,6 +14,7 @@ class MockDefaultsStore: DefaultsStorable {
     }
     
     func removeObject(forKey defaultName: String) {
+        self.didCallRemoveObject = true
```

Comment:
I imagine you could probably just remove the object set in the test: `mockDefaultStore.set(Date() + 60, forKey: .accessTokenExpiry)` and assert that that is nil?

Reply:
Would it be better to check that all of `mockDefaultStore`'s `savedData` is empty or to specifically assert that the access token expiry is nil?